src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -11,19 +11,50 @@
     <DocumentationFile>$(OutputPath)\DynamoCoreWpf.XML</DocumentationFile>
     <EnableDefaultCompileItems>False</EnableDefaultCompileItems>
     <EnableDefaultEmbeddedResourceItems>False</EnableDefaultEmbeddedResourceItems>
+    <PreferredToolArchitecture>x64</PreferredToolArchitecture>
   </PropertyGroup>
 
   <Target Name="NpmRunBuild" BeforeTargets="BeforeBuild">
-      <Exec Command="npm install --prefix ./ @dynamods/splash-screen@latest" />
+      <!--This command gets the latest build from the splash screen-->
+      <Exec Command="npm pack @dynamods/splash-screen@latest" />
   </Target>
-  <ItemGroup>
-    <None Remove="node_modules\%40dynamods\splash-screen\build\index.bundle.js" />
-    <None Remove="node_modules\%40dynamods\splash-screen\build\index.html" />
+
+    <Target Name="ExtractTGZFile" DependsOnTargets="NpmRunBuild" BeforeTargets="BeforeBuild">        
+        <!--Locates the .tgz files-->
+        <ItemGroup>
+            <TGZFiles Include="./dynamods-splash-screen-*.tgz" />
+        </ItemGroup>
+
+        <!--Reverse the order of the files to get the higher version-->
+        <ItemGroup>
+            <Reversed Include="@(TGZFiles-&gt;Reverse())" />
+        </ItemGroup>
+        
+        <PropertyGroup>
+            <Last>%(TGZFiles.Filename)</Last>
+        </PropertyGroup>
+
+        <!--Extracts the file to /package-->
+        <Exec Command="tar -xzf $(MSBuildProjectDirectory)\$(Last).tgz"></Exec>
+
+        <!--Deletes the tgz file-->
+        <Delete Files="$(MSBuildProjectDirectory)\$(Last).tgz" />
+    </Target>
+
+
+    <ItemGroup>
+        <PackageReference Include="7-Zip.StandaloneConsole.x64" Version="22.1.0" />
+    </ItemGroup>
+
+
+    <ItemGroup>
+    <None Remove="package\build\index.bundle.js" />
+    <None Remove="package\build\index.html" />
     <None Remove="Views\SplashScreen\WebApp\splashScreenBackground.png" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="node_modules\@dynamods\splash-screen\build\index.bundle.js" />
-    <EmbeddedResource Include="node_modules\@dynamods\splash-screen\build\index.html" />
+    <EmbeddedResource Include="package\build\index.bundle.js" />
+    <EmbeddedResource Include="package\build\index.html" />
```

Comment:
If there is no need for a specific package name, does it mean that we would only support a single npm package per project? Do we need to specify the folder name during unzipping?

Reply:
Is not necessary to specify the folder name during unzipping. By default, the folder created is called _package_. We can create folders named properly using the _--directory_ argument from tar.

Reply:
@filipeotero Understood, my point is that if in the future `DynamoCoreWpf` is depending on multiple npm packages, they will all be unzipped to the `package` folder and the index.html will be overwriting each other. If possible, I would love to have the dir specified to prevent that case from happening, what do you think?

Reply:
Makes sense. I will do that!